src/Tracer.cpp

```diff
@@ -12,17 +12,24 @@
 
 
 
+const float FLOAT_EPSILON = 0.0001f;  // TODO: Stash this in some header where it can be reused
+
+
+const std::array<const Vector3f, 6> cTracer::m_NormalTable =
+{{
```

Comment:
You need to put each brace on a separate line. The basic style checker has a false positive for this occurrence.